ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.ts

```diff
@@ -0,0 +1,482 @@
+        },
+        getBehindParty: {
+          en: 'Behind party',
+        },
+        getBehindPartyKnockback: {
+          en: 'Behind party (Knockback)',
+        },
+        getInFrontOfPlayer: {
+          en: 'In front of ${player}',
+        },
+        getInFrontOfPlayerKnockback: {
+          en: 'In front of ${player} (Knockback)',
+        },
+      },
+    },
+    {
+      id: 'ASS Slippery Soap with Chilling Suds',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '775E', source: 'Silkie' }),
+      condition: (data) => data.suds === '7758',
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 1,
+      response: Responses.moveAround(),
+    },
+    {
+      id: 'ASS Slippery Soap After',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '775E', source: 'Silkie', capture: false }),
+      infoText: (data, _matches, output) => {
+        switch (data.suds) {
+          case '7757':
+            return output.getUnder!();
+          case '7758':
+            return output.intercards!();
+          case '7759':
+            return output.spreadCardinals!();
+        }
+      },
+      outputStrings: {
+        getUnder: Outputs.getUnder,
+        spreadCardinals: {
+          en: 'Spread Cardinals',
+        },
+        intercards: {
+          en: 'Intercards',
+          de: 'Interkardinal',
+          fr: 'Intercardinal',
+          ja: '斜めへ',
+          cn: '四角',
+          ko: '대각선 쪽으로',
+        },
+      },
+    },
+    {
+      id: 'ASS Carpet Beater',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '774F', source: 'Silkie' }),
+      preRun: (data) => data.beaterCounter++,
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          busterOnYou: Outputs.tankBusterOnYou,
+          busterOnTarget: Outputs.tankBusterOnPlayer,
+          busterOnYouPuffs: {
+            en: 'Tank Buster on YOU, East/West Between Puffs ',
+          },
+        };
+
+        if (matches.target === data.me) {
+          if (data.beaterCounter === 2)
+            return { alertText: output.busterOnYouPuffs!() };
+          return { infoText: output.busterOnYou!() };
+        }
+
+        if (data.role !== 'tank' && data.role !== 'healer')
+          return;
+
+        return { infoText: output.busterOnTarget!({ player: data.ShortName(matches.target) }) };
+      },
+    },
+    {
+      id: 'ASS Total Wash',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '7750', source: 'Silkie', capture: false }),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'aoe + bleed',
+          de: 'AoE + Blutung',
+          fr: 'AoE + Saignement',
+          ja: 'AOE + 出血',
+          ko: '전체 공격 + 도트',
+        },
+      },
+    },
+    // ---------------- Gladiator of Sil'dih ----------------
```

Comment:
```suggestion
    // ---------------- second trash  ----------------
    {
      id: 'ASS Infernal Pain',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['7969', '7981'], source: 'Sil\'dihn Dullahan', capture: false }),
      infoText: (_data, _matches, output) => output.text!(),
      outputStrings: {
        text: {
          en: 'aoe + bleed',
          de: 'AoE + Blutung',
          fr: 'AoE + Saignement',
          ja: 'AOE + 出血',
          ko: '전체 공격 + 도트',
        },
      },
    },
    {
      id: 'ASS Blighted Gloom',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['7966', '797E'], source: 'Sil\'dihn Dullahan', capture: false }),
      response: Responses.getOut(),
    },
    {
      id: 'ASS King\'s Will',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['7968', '7980'], source: 'Sil\'dihn Dullahan', capture: false }),
      infoText: (_data, _matches, output) => output.text!(),
      outputStrings: {
        text: {
          en: 'big autos',
        },
      },
    },
    {
      id: 'ASS Hells\' Nebula',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['796C', '7984'], source: 'Aqueduct Armor', capture: false }),
      infoText: (_data, _matches, output) => output.text!(),
      outputStrings: {
        text: {
          en: 'HP to 1',
        },
      },
    {
      id: 'ASS Infernal Weight',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['796B', '7983'], source: 'Aqueduct Armor', capture: false }),
      infoText: (_data, _matches, output) => output.text!(),
      outputStrings: {
        text: {
          en: 'aoe + heavy',
        },
      },
    },
    {
      id: 'ASS Dominion Slash',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['796A', '7982'], source: 'Aqueduct Armor', capture: false }),
      response: Responses.getBehind(),
    },
    // ---------------- Gladiator of Sil'dih ----------------
```

Reply:
I think there are similar triggers for handling a mechanic such ad King's Will and Hell's Nebula. I'm not sure if a mechanic such as Infernal's Weight has existed. Infernal's Weight I think targets a random player as well, so might that be accounted for where the followup Cleave is going.

Reply:
From my logs Infernal Weight targets everyone (no specific target), and the follow-up Dominion Slash has no target, so I don't know if there's a possibility of calling where it's going without looking at mob facing.

Reply:
Ah, well we can adjust the domion slash later if necessary.
I think the Infernal Weight will be better with a aoe + getin call instead so that the followup Slash can be safely avoided.

I would remove the Hell's Nebula trigger, cactbot historically does not call out for these unless there is a requirement to heal up after (often that is a trigger for the spell that requires health threshold and not the white hole spell itself). I'm not sure an exception should be made here as you could potentially be pulling multiple mobs at once. Examples: E12S Hell's Judgment, O4S White Hole.

I would also remove the King's Will trigger. I believe these spells have been reviewed in previous encounters, but no trigger specific to the buff was made? Effectively this buff does behave similarly to a mini tankbuster (albeit multiple ones for the duration) which we have had an instance of in a dungeon boss before, I think it was a Shadowbringers dungeon. So maybe change to something like that if kept I would change to a gainsEffect trigger and set duration to match the buffs timer.

Reply:
Done.  I may circle back on the King's Will trigger later as a tank/heal only heads-up.

Reply:
Thanks!